From 05385ad17cd605f3ac26081224dff3d35fa176d1 Mon Sep 17 00:00:00 2001
From: Alexander Forsch <alexander.forsch@bytesatwork.ch>
Date: Mon, 11 Nov 2024 13:39:55 +0100
Subject: [PATCH] arm64: freescale: imx93-bytedevkit.dts: Fix touch and display
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Adapt pins for lcd display and touch sensor with byteENGINE IMX93 OSM 2.0
on byteDEVKIT OSM IMX93 1.0.

Note: Previous revisions are no longer supported.

Signed-off-by: Alexander Forsch <alexander.forsch@bytesatwork.ch>
Signed-off-by: Stephan Dünner <stephan.duenner@bytesatwork.ch>
---
 arch/arm64/boot/dts/freescale/imx93-bytedevkit.dts | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/imx93-bytedevkit.dts b/arch/arm64/boot/dts/freescale/imx93-bytedevkit.dts
index 0eacfec94180..3d01ef03ba90 100644
--- a/arch/arm64/boot/dts/freescale/imx93-bytedevkit.dts
+++ b/arch/arm64/boot/dts/freescale/imx93-bytedevkit.dts
@@ -197,9 +197,9 @@ gt911@5d {
 		compatible = "goodix,gt911";
 		reg = <0x5d>;
 		interrupt-parent = <&gpio2>;
-		interrupts = <23 0>;
-		irq-gpios = <&gpio2 23 0>;
-		reset-gpios = <&gpio2 24 0>;
+		interrupts = <21 0>;
+		irq-gpios = <&gpio2 21 0>;
+		reset-gpios = <&gpio2 20 0>;
 	};
 };
 
@@ -279,7 +279,7 @@ panel@0 {
 		compatible = "youritech,ili9806e";
 		reg = <0>;
 		backlight = <&mipi_bl>;
-		reset-gpio = <&gpio2 22 GPIO_ACTIVE_HIGH>;
+		reset-gpio = <&gpio2 10 GPIO_ACTIVE_HIGH>;
 		dsi-lanes = <2>;
 		video-mode = <2>;	/* 0: burst mode
 					 * 1: non-burst mode with sync event
@@ -494,15 +494,15 @@ MX93_PAD_GPIO_IO21__GPIO2_IO21		0x31e	/* LVDS Touch RST */
 
 	pinctrl_mipi_touch: mipi_touch_grp {
 		fsl,pins = <
-			MX93_PAD_GPIO_IO23__GPIO2_IO23		0x31e	/* MIPI Touch INT */
-			MX93_PAD_GPIO_IO24__GPIO2_IO24		0x31e	/* MIPI Touch RST */
+			MX93_PAD_GPIO_IO21__GPIO2_IO21		0x31e	/* MIPI Touch INT */
+			MX93_PAD_GPIO_IO20__GPIO2_IO20		0x31e	/* MIPI Touch RST */
 		>;
 	};
 
 	pinctrl_mipi_blacklight: mipi_bl_grp {
 		fsl,pins = <
 			MX93_PAD_CCM_CLKO1__GPIO3_IO26		0x31e	/* MIPI Backlight */
-			MX93_PAD_GPIO_IO22__GPIO2_IO22		0x31e	/* MIPI Reset */
+			MX93_PAD_GPIO_IO10__GPIO2_IO10		0x31e	/* MIPI Reset */
 		>;
 	};
 };
